cobbler/actions/mkloaders.py

```diff
@@ -69,13 +79,22 @@ def run(
                 self.bootloaders_dir.joinpath("grub", bl_mod_dir),
             )
 
+    def create_directories(self):
+        """
+        Create the required directories so that this succeeds. If existing, do nothing.
+        """
+        if not self.bootloaders_dir.exists():
+            raise FileNotFoundError("Main bootloader directory not found! Please create it yourself!")
+
+        grub_dir = self.bootloaders_dir.joinpath("grub")
+        if not grub_dir.exists():
+            os.mkdir(grub_dir, 0o644)
+
 
 # NOTE: move this to cobbler.utils?
 # cobbler.utils.linkfile does a lot of things, it might be worth it to have a
 # function just for symbolic links
-def symlink(
-    target: pathlib.Path, link: pathlib.Path, skip_existing: bool = False
-) -> None:
+def symlink(target: pathlib.Path, link: pathlib.Path, skip_existing: bool = False):
```

Comment:
Just wondering, why do you remove type annotations here?

Reply:
Because we return nothing, so no type annotation needed afaik. And yes I am aware that this implicitly means we do return `None`.

Reply:
There is no such thing as returning nothing, you always return `None` or something else. I don't mind this policy, having no type annotation for the return value if there is no `return` statement in the body, but then I would extend it to the docstring as well.

Reply:
Well the docstring is autogenerated, so if they generate that for us I wouldn't put work into that. Or do you mean that if a variable is `None` because I am not aware of any variables which we require to be `None` (does that even make sense)?

Reply:
The docstring is autogenerated? What's is taking care of that? I used sphinx-doc-mode, is there something IDE-independent that you use?

Reply:
I just use my PyCharm integration of Sphinx. So sadly nothing IDE independent. It even tries to analyze the code and guess the return type in case it is not given. This is in 90% of complex cases fully correct and in 100% of the easy ones. But I guess this will go away with blacked too?